packages/video_player/video_player_android/android/build.gradle

```diff
@@ -48,11 +48,12 @@ android {
     }
 
     dependencies {
-        def exoplayer_version = "2.18.7"
-        implementation "com.google.android.exoplayer:exoplayer-core:${exoplayer_version}"
-        implementation "com.google.android.exoplayer:exoplayer-hls:${exoplayer_version}"
-        implementation "com.google.android.exoplayer:exoplayer-dash:${exoplayer_version}"
-        implementation "com.google.android.exoplayer:exoplayer-smoothstreaming:${exoplayer_version}"
+        def exoplayer_version = "1.1.1"
+        implementation "androidx.media3:media3-exoplayer:${exoplayer_version}"
+        implementation "androidx.media3:media3-exoplayer-hls:${exoplayer_version}"
+        implementation "androidx.media3:media3-exoplayer-dash:${exoplayer_version}"
+        implementation "androidx.media3:media3-exoplayer-smoothstreaming:${exoplayer_version}"
+        debugImplementation "androidx.annotation:annotation-experimental:1.3.0"
```

Comment:
`androidx.media3:media3-common:1.1.1` depends on `androidx.annotation:annotation-experimental:1.2.0`

however annotation-experimental 1.2.0 leads to lint failure:
```
jetified-annotation-experimental-1.2.0/jars/lint.jar: Error: Library lint checks out of date;
```

so, since lintDebug only is used - bump annotation-experimental to 1.3.0 in debug builds

media3 uses annotation-experimental 1.3.1 since https://github.com/androidx/media/releases/tag/1.2.0, and this could be removed later

Reply:
What do you mean by removing it later? Is there a reason we shouldn't use it? Also, if we bump to 1.3.1 anyway, that will align.